public/js/notes.js

```diff
@@ -11,24 +11,24 @@ function updateNote() {
             <th scope="row">${index + 1}</th>
             <td>${element[0]}</td>
             <td>${element[1]}</td>
-            <td><button id="dBtn" onclick="deleteNote(${index});">Delete</button></td>
+            <td><button id="dBtn" onclick="deleteNotes(${index});">Delete</button></td>
             </tr>`;
     });
   } else {
     console.log("Error in adding notes");
   }
   tableBody.innerHTML = str;
 }
-updateNote(); //calling update to update table
+fetchNotes(); //calling update to update table
 
 //deletes the data from table and localstorage
-function deleteNote(itemIndex) {
+function deleteNotes(itemIndex) {
```

Comment:
Are you deleting a note based on "itemIndex"?

Reply:
Wrong function name, if your function is deleting 1 entry based on provided "itemIndex" then it name should be "deleteNote" and if deleting multiple based on array of indexes then it's "deleteNotes". Function name should be meaningful.